tools/pythonpkg/src/python_objects.cpp

```diff
@@ -118,10 +120,12 @@ void PyDecimal::SetExponent(py::handle &exponent) {
 	ExponentNotRecognized();
```

Comment:
It seems no test triggers this function call, can we have a test that triggers it?